File: python3/Contains_Duplicate.py
/*
Input: nums = [1,2,3,1]
Output: true

Input: nums = [1,2,3,4]
Output: false
*/

class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        index = set(nums)
        if len(nums) > len(index):
            return True
        else:
            return False

File: C/Contains_Duplicate.c
/*
Input: nums = [1,2,3,1]
Output: true

Input: nums = [1,2,3,4]
Output: false
*/

bool containsDuplicate(int* nums, int numsSize){

    int result, outerloop, innerloop;
    int* local = nums;
    for(outerloop = numsSize-1;outerloop>=0;outerloop--)
    {
        for(innerloop = outerloop;innerloop>=0;innerloop--)
        {
            result = local[outerloop] ^ local[innerloop-1];
            if(result == 0)
                return true;
        }
    }
    
    return false;
}
